src/index.js

```diff
@@ -28,94 +28,96 @@ function init( {
 	events = {},
 	debug = false
 } ) {
-	const globalLang = lang,
-		popup = isTouch ?
-			createTouchPopup( popupContainer ) :
-			createPopup( popupContainer ),
-		popupEvents = customEvents( popup ),
-		last = {},
-		foundSelectorLinks = [],
-		foundDetectLinks = [],
-		showPopup = ( e, refresh = false ) => {
-			e.preventDefault()
-
-			const popupId = Date.now(),
-				{ target } = refresh ? last : e,
-				title = refresh ? last.title : target.getAttribute( 'data-wp-title' ) || target.textContent,
-				lang = refresh ? last.lang : target.getAttribute( 'data-wp-lang' ) || globalLang,
-				pointerPosition = refresh ? last.pointerPosition : { x: e.clientX, y: e.clientY },
-				dir = getDir( lang )
-
-			if ( popup.element.currentTargetElement === target && !refresh ) {
-				// Hovering over the same link and the popup is already open
-				return
-			}
+	const globalLang = lang
+	const popup = isTouch ?
+		createTouchPopup( popupContainer ) :
+		createPopup( popupContainer )
+	const popupEvents = customEvents( popup )
+	const last = {}
+	const foundSelectorLinks = []
+	const foundDetectLinks = []
+
+	const showPopup = ( e, refresh = false ) => {
+		e.preventDefault()
+
+		const popupId = Date.now()
+		const { target } = refresh ? last : e
+		const title = refresh ? last.title : target.getAttribute( 'data-wp-title' ) || target.textContent
+		// eslint-disable-next-line no-shadow
```

Comment:
`lang` is technically declared already from method arguments. Let me know if we want to update the name of one or other, it'd likely have repercussions in other parts of the code

Reply:
We should fix that by renaming `lang` to something else so it doesn't shadow the `lang` argument from the outer function.

Reply:
How about `localLang` to parallel the `globalLang` we already have in there? See https://github.com/wikimedia/wikipedia-preview/pull/112/commits/b4fde023b472ec1d9522ee1887cd8f6799e27ae6